dotnet/src/Extensions/Extensions.UnitTests/Planning/ActionPlanner/ActionPlannerTests.cs

```diff
@@ -1,12 +1,15 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+using System;
 using System.Collections.Generic;
+using System.Text;
```

Comment:
`System.Text` is not needed here, that's why build pipeline is failing:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/57047239-ac49-47b1-a7ac-9739f8f3b20d)

Before commit or PR creation you can run `dotnet/build.sh` or `dotnet/build.cmd`, which should resolve such issues automatically.

Reply:
Thanks!